cluster/addons/addon-manager/Makefile

```diff
@@ -15,10 +15,10 @@
 IMAGE=gcr.io/k8s-staging-addon-manager/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v9.1.7
-KUBECTL_VERSION?=v1.27.1
+VERSION=v9.1.8
+KUBECTL_VERSION?=v1.30.0
```

Comment:
Just for my understanding.. any particular reason to use 1.30 (and not newer)?

Reply:
Do you mean specifically why 1.30.0 (the .0 over 1.30.1) or why not 1.31? 1.31 isn't released yet and cannot be committed, I do see 1.30.2 as the latest version that we could use so I could update to that.

Reply:
:+1: 1.30.2 SGTM

Reply:
fixed.